--- a/dictionary/user_dictionary.h
+++ b/dictionary/user_dictionary.h
@@ -15,7 +15,7 @@

 struct StenoUserDictionaryData {
   StenoUserDictionaryData() = default;
-  constexpr StenoUserDictionaryData(const uint8_t *mem, size_t size) {
+  StenoUserDictionaryData(const uint8_t *mem, size_t size) {
     assert((size & (size - 1)) == 0);
     hashTable = (uint32_t *)mem;
     reverseHashTable = (const uint32_t *)(mem + size / 8);
